packages/editor-ui/src/components/mixins/workflowHelpers.ts

```diff
@@ -613,5 +635,8 @@ export const workflowHelpers = mixins(
 
 				return true;
 			},
+			getNodeName(nodeName: string) {
+				return nodeName.replace(/ trigger/i, '');
+			},
```

Comment:
can you make this into a stateless function instead in a `helpers.ts` file? for one, this function does not require any state so we should avoid forcing it to use state. second, mixins are terrible and we should avoid using them more... they make it hard to know what a class is made of and also create name conflicts..

i would also rename this to something a lot more specific. as you will quickly realise `name` and `nodeName` are two very confusing aspects of our setup